### ISSUE TITLE
Inherit milestone from issue

### ISSUE THREAD

**Issue by t-izbassar** [yegor256/0pdd#174]
Scenario: issue 555 have milestone 2.0 assigned. When I work on that issue and create puzzles, those new issues should have the same 2.0 milestone assigned from the farm issue because it is obvious that puzzles are part of the issue and as issue is targeted to 2.0 release those puzzles should get the same target.

So the proposition is: get the milestone from issue if it's present and assign it to new task. This behavior should also be configurable. Something like that:
`- inherit_milestone: true`

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@t-izbassar totally agree, thanks for reporting!

**Comment by 0crat** [comment]
Job [#174](https://github.com/yegor256/0pdd/issues/174) is now in scope, role is `DEV`

**Comment by php-coder** [comment]
> This behavior should also be configurable.

Yes, please. Because in my case all of the creates issues have "techdebt" label and they shouldn't belong to the any milestone.

**Comment by 0crat** [comment]
The job [#174](https://github.com/yegor256/0pdd/issues/174) assigned to @rok-povsic[/z](https://www.0crat.com/u/rok-povsic), here is [why](https://www.0crat.com/footprint/C3T46CUJJ/521912865). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by 0crat** [comment]
@rok-povsic[/z](https://www.0crat.com/u/rok-povsic) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by rok-povsic** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/0pdd/issues/174#issuecomment-369504908))

@rok-povsic The user @rok-povsic[/z](https://www.0crat.com/u/rok-povsic) resigned from [#174](https://github.com/yegor256/0pdd/issues/174), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 points just awarded to @rok-povsic[/z](https://www.0crat.com/u/rok-povsic)

**Comment by 0crat** [comment]
The job [#174](https://github.com/yegor256/0pdd/issues/174) assigned to @silasreinagel[/z](https://www.0crat.com/u/silasreinagel), here is [why](https://www.0crat.com/footprint/C3T46CUJJ/1130937998); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by SilasReinagel** [comment]
@0crat wait. Blocked by #200

**Comment by 0crat** [comment]
> @0crat wait. Blocked by #200  ([here](https://github.com/yegor256/0pdd/issues/174#issuecomment-384503357))

@SilasReinagel Can't understand "wait.", try one of these:
  - `assign` Assign a performer to this issue
  - `boost` Set a boost for the job
  - `hello` Just say hello
  - `in` Register this issue in scope (WBS)
  - `out` Close the order and remove this job from scope
  - `pay` Pay a user some extra cash
  - `quality` Review a task
  - `resign` Resign from this issue
  - `status` Check the status of the job
  - `version` Print current version of the bot
  - `wait` Register an impediment for a job

**Comment by SilasReinagel** [comment]
@0crat wait

**Comment by 0crat** [comment]
> @0crat wait ([here](https://github.com/yegor256/0pdd/issues/174#issuecomment-384503453))

@SilasReinagel The impediment for [#174](https://github.com/yegor256/0pdd/issues/174) was registered successfully by @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Comment by llorllale** [comment]
@yegor256 @SilasReinagel I'd really like this one implemented...

Can I contribute with a PR?

**Comment by SilasReinagel** [comment]
@llorllale Yes. If you create a PR that solves this, you can have the issue.

**Comment by llorllale** [comment]
@t-izbassar @SilasReinagel I submitted PR #229 for this

**Comment by llorllale** [comment]
@t-izbassar PR#229 was merged. Can this issue be closed?

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Comment by 0crat** [comment]
The job [#174](https://github.com/yegor256/0pdd/issues/174) is now out of scope
